### ISSUE TITLE
PEP 541 Request: `pybabel`

### ISSUE THREAD

**Issue by cfm** [pypi/support#3218]
### Project to be claimed

`pybabel`: https://pypi.org/project/pybabel

### Your PyPI username

none

### Reasons for the request

1. `Babel` (https://pypi.org/project/Babel) provides `pybabel`.
2. `pybabel` (https://pypi.org/project/pybabel) provides `pbc`, but the published package is at version `0.0.0.dev0`.

(2) isn't invalid per se, but it conflicts with (1), which leads to unnecessary surprises when `pip install pybabel` gets you something completely unrelated.

### Maintenance or replacement?

None

### Source code repositories URLs

| Package | Repository | Repository last updated |
| --- | --- | --- |
| https://pypi.org/project/Babel/ | https://github.com/python-babel/babel | 27 July 2023 |
| https://pypi.org/project/pybabel/ | https://github.com/pybpc/bpc | 22 November 2020 |

### Contact and additional research

I'd be happy to contact the owners, but my interest here is in disambiguating the PyPI ecosystem, not in making any package name available for my own use.

### Code of Conduct

- [X] I agree to follow the PSF Code of Conduct

**Comment by Thespi-Brain** [comment]
Hi @cfm,

We are working to contact the owner to determine if they're reachable and have sent them a courtesy message.

Final determination regarding project ownership will be made by the PyPI Administrators when the research process is complete.

**Comment by Thespi-Brain** [comment]
**pybabel** has been deemed invalid. Additionally, the owner did not respond back regarding the status of their project. 
I recommend that the PyPI Admins delete the project and release the name for future use.

**Comment by ewdurbin** [comment]
Hi @cfm. What action are we looking for at this point? You filed this issue as `None` for Maintenance or Replacement. Is the goal to see the name pybabel taken down and prohibited from future registration?

**Comment by cfm** [comment]
> Is the goal to see the name pybabel taken down and prohibited from future registration?

Yes, exactly, in deference to the `pybabel` provided by the `Babel` package.

**Comment by ewdurbin** [comment]
Deleted the project 'pybabel'

Prohibited Project Name 'pybabel'
